### PR TITLE
Fix linting and format ows_cfg

### PR DESCRIPTION
**Pull request by pindge**
```
Run npm install jsonlint -g
npm WARN deprecated nomnom@1.8.1: Package no longer supported. Contact support@npmjs.com for more info.
npm WARN checkPermissions Missing write access to /usr/local/lib/node_modules
npm ERR! code EACCES
npm ERR! syscall access
npm ERR! path /usr/local/lib/node_modules
npm ERR! errno -13
npm ERR! Error: EACCES: permission denied, access '/usr/local/lib/node_modules'
npm ERR!  [Error: EACCES: permission denied, access '/usr/local/lib/node_modules'] {
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'access',
npm ERR!   path: '/usr/local/lib/node_modules'
npm ERR! }
npm ERR! 
npm ERR! The operation was rejected by your operating system.
npm ERR! It is likely you do not have the permissions to access this file as the current user
npm ERR! 
npm ERR! If you believe this might be a permissions issue, please double-check the
npm ERR! permissions of the file and its containing 
```

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -29,10 +29,11 @@ jobs:
 
     - name: Run Json Linter
       run: |
-        npm install jsonlint -g
-        jsonlint dev/terria/dea-maps.json
+        sudo npm install jsonlint -g
```
Comment (whatnick) [inline .github/workflows/main.yml:32]: Don't need sudo if NPM has been setup by Github actions.

**Review by whatnick — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (pindge) [reply .github/workflows/main.yml:32]: unless something else has changed, but without sudo it doesn't work, we can try and see if it works this time

Comment (pindge) [reply .github/workflows/main.yml:32]: image
all those fails eventually successed with `sudo`

Comment (whatnick) [reply .github/workflows/main.yml:32]: Ya need github actions to install separate NPM instead of using stock NPM, see master.

**Review by whatnick — APPROVED** [review]

**Pull request merged**